dev/preview/previewctl/BUILD.yaml

```diff
@@ -20,32 +20,11 @@ packages:
     config:
       commands:
         - [ "sh", "-c", "mkdir -p /workspace/bin && sudo mv dev-preview-previewctl--cli/previewctl /workspace/bin/previewctl" ]
-        - [ "sh", "-c", "if ! $(grep 'previewctl completion bash' ~/.bashrc > /dev/null); then previewctl completion bash >> ~/.bashrc; fi" ]
+        - [ "sh", "-c", "if ! $(grep 'previewctl completion bash' ~/.bashrc > /dev/null); then echo '. <(previewctl completion bash)' >> ~/.bashrc; fi" ]
+        - [ "sh", "-c", "if ! $(grep 'previewctl completion zsh' ~/.zshrc > /dev/null); then echo '. <(previewctl completion zsh)' >> ~/.zshrc; fi" ]
 
 scripts:
   - name: install
     description: Build and install previewctl into the current environment
-    deps:
-    - :cli
-    script: |
-      set -euo pipefail
-
-      destination="/workspace/bin/previewctl"
-      # Leeway puts the dependencies at the end of the PATH which means that /workspace/bin takes precedence
-      # on the path. So for `which previewctl` to return the previewctl Leeway just build (/tmp/build/.../previewctl)
-      # we have to delete /workspace/bin/previewctl first.
-      rm -f "$destination"
-      source="$(which previewctl)"
-
-      mkdir -p /workspace/bin
-
-      echo "Installing $source into $destination"
-      cp "$source" "$destination"
-
-      if ! $(grep 'previewctl completion bash' ~/.bashrc > /dev/null)
-      then
-        echo "Adding bash completions to ~/.bashrc"
-        echo '. <(previewctl completion bash)' >> ~/.bashrc
-      else
-        echo "Bash completions already installed. Skipping."
-      fi
+    script:
+      leeway build dev/preview/previewctl:install -Dversion=$RANDOM --dont-test --cache=remote-pull
```

Comment:
I we use "--dont-test" here then I don't believe we'll ever run the tests as part of CI as we're not otherwise building the package AFAIK. Maybe add it to `dev:all-app` so it gets included in the default `components:all` package.

Reply:
Why is specifying "remote-pull" for the cache needed?

Reply:
> Maybe add it to `dev:all-app`

✅ 

> Why is specifying "remote-pull" for the cache needed?

It build the package without pushing the artifact to the remote, as it's not necessary.

Reply:
> Builds the package without pushing the artifact to the remote, as it's not necessary.

If we don't push there, when is the packages ever pushed then? I understand that `dev/preview/previewctl:install` doesn't need to be pushed, but the packages it depends on should (e.g. dev/preview/previewctl:cli). This might be less important now that it's included in `dev:all-app` though ☺️

Reply:
From `dev:all-app` should be enough. I missed that we hadn't added it anywhere before, but now that it's there I don't think we need anything else. Doing changes to previewctl will persist in the local cache and once the build runs in CI it will be available globally.